docs/semgrep-supply-chain/findings.md

```diff
@@ -0,0 +1,137 @@
+
+:::info Prerequisite
+At least one repository that scans for dependencies through Semgrep Supply Chain. See [Scan third-party dependencies](/semgrep-supply-chain/getting-started).
+:::
+
+## View findings
+
+To view your findings in Semgrep AppSec Platform:
+
+1. Log in to [Semgrep AppSec Platform](https://semgrep.dev/login).
+2. In the **Navigation bar**, click **[Supply Chain](https://semgrep.dev/orgs/-/supply-chain)**.
+
+By default, Semgrep displays your **Priority** findings. Priority findings are defined as findings that:
+
+- Have a severity level of **critical** or **high**
+- Are **reachable**
+
+You can switch to the **All** tab at any point to view all findings identified by Semgrep Supply Chain. Both the **Priority** findings view and the **All** findings view display high-level information about your findings.
+
+## Filter findings
+
+Regardless of whether you use the **Priority** findings view or the **All** findings view, there are multiple grouping and filtering options available to you.
+
+### Time period
+
+<TimePeriodFilters />
+
+### Project
+
+The **Project** filter allows you to search for findings associated with the selected projects.
+
+### Status
+
+The **Status** filter allows you to search for findings in the selected statuses, which are the triage states of the findings:
+
+* **Open**: Findings for which there have been no triage or remediation action.
+* **Reviewing**: Findings that require more investigation to determine what the next steps should be.
+* **To fix**: Findings for which you have decided to fix. Commonly used to indicate that these findings are tracked in Jira or assigned to developers for further work.
+* **Provisionally ignored**: Findings that Semgrep Assistant has flagged as false positives. You can change the status to **Ignored** if you agree with Assistant's assement. Otherwise, you can change the status to **To fix** if you don't agree.
+* **Ignored**: Vulnerabilities that have been triaged as **Ignored** by the user. You can filter findings with a status of **Ignored** further by reason: **False positive**, **Acceptable risk**, **No time to fix**, or **No triage reason**.
+* **Closed**: Vulnerabilities that are no longer detected after a scan. This typically means that the dependency containing the vulnerability has been updated. Semgrep Supply Chain automatically checks if the dependency has been updated and sets the vulnerability's status as **Fixed**.
+
+> You can set the **Fixing** and **Reviewing** statuses only if you are a [Jira beta](https://semgrep.dev/docs/semgrep-appsec-platform/jira) participant.
```

Comment:
Is Jira integration still in beta? The Jira int page is tagged with beta, but it doesn't say beta anywhere else.